BDArmory/Control/MissileFire.cs

```diff
@@ -2473,8 +2473,9 @@ IEnumerator GuardBombRoutine()
             guardFiringMissile = true;
             float bombStartTime = Time.time;
             float bombAttemptDuration = Mathf.Max(targetScanInterval, 12f);
-            float radius = CurrentMissile.GetBlastRadius() * Mathf.Min(maxMissilesOnTarget / 2f, 1.5f);
-            MissileLauncher cm = CurrentMissile as MissileLauncher;
+            float radius = CurrentMissile.GetBlastRadius() * Mathf.Max(0.68f + 1.4f * (maxMissilesOnTarget - 1f), 1.5f);
+            var cb = CurrentMissile.GetPart(). FindModuleImplementing<ClusterBomb>();
+            if (cb != null) { radius *= Mathf.Max(0.68f * cb.submunitions.Count(), 1f); }
```

Comment:
missileBase already has an ``public int clusterbomb`` field (default 1) counting number of submunitions that could optimize this a bit.